### PR TITLE
Tech News Site

### PR DESCRIPTION
**Pull request by savannah-hayes**

### CODE REVIEW

**Review by rawisou — APPROVED** [review]
**Website functionality**
Your website works as intended. No bugs.
**Code** 
Your code is very easy to read and understand. I like the naming system that you in both classes and variables. I gave a comment about the `main `tag in html but I don't think this requires any changes. Please let me know if there is a reason behind it. 

Great job! I am inspired by your code and how you write it!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        </li>
+      </ul>
+      <div class="navbar-toggle">
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+      </div>
+    </nav>
+  </header>
+  <main class="main">
```
[the remark this answers is not in the document]
Comment (savannah-hayes) [reply index.html:46]: Yes, that is a great comment. I have removed the class now as it is repetitive.

File: assets/javascript/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const hamburgerMenu = document.querySelector(".navbar-toggle");
+const ul = document.querySelector(".navbar__list");
+const nav = document.querySelector(".navbar");
+const navLinks = document.querySelectorAll(".navbar__item");
```
[the remark this answers is not in the document]
Comment (savannah-hayes) [reply assets/javascript/index.js:4]: Thank you :)

File: assets/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,290 @@
+:root {
```
[the remark this answers is not in the document]
Comment (savannah-hayes) [reply assets/styles/style.css:1]: This looks very similar to sass but it is actually something called CSS custom properties (variables). This is a great way to reuse specific values throughout a document.

**Comment by savannah-hayes** [comment]
I have made the suggested changes to my code and want to thank you for your code review.

**Pull request closed without merging**